src/libcore/intrinsics.rs

```diff
@@ -553,28 +553,148 @@ extern "rust-intrinsic" {
     /// [`AtomicBool::fetch_xor`](../../std/sync/atomic/struct.AtomicBool.html#method.fetch_xor).
     pub fn atomic_xor_relaxed<T>(dst: *mut T, src: T) -> T;
 
+    /// Maximum with the current value using a sized comparison.
+    /// The stabilized version of this intrinsic is available on the
```

Comment:
there needs to be an empty `///` between the summary sentence and the start of the rest of the docs, like this:

```
/// Maximum with the current value using a sized comparison.
///
/// The stabilized...
```

Reply:
There are a lot of (already existing) intrinsics documentations that do not follow this rule. Do you want me to go through and fix them all?